README.md

```diff
@@ -34,16 +34,10 @@ Here's how to start zipkin using the default file-based backend and view traces.
 ```bash
 # get the zipkin source and change to its directory
 $ git clone https://github.com/openzipkin/zipkin; cd zipkin
-# start the query server in a new terminal session or tab
+# start the query server
 $ ./bin/query
-# start the collector server in a new terminal session or tab
-$ ./bin/collector
-# start the web server in a new terminal session or tab
-$ ./bin/web
-# create dummy traces
-$ ./bin/tracegen
```

Comment:
spigo demos always use docker, so I've moved this to a new issue https://github.com/openzipkin/docker-zipkin-java/issues/6 I think there are some ways we can help with spigo to reduce steps (possibly making it report directly as opposed to generating files similar to https://github.com/adrianco/spigo/issues/67)